rotkehlchen/tests/unit/globaldb/test_migrations.py

```diff
@@ -18,7 +18,7 @@ def test_migration1(globaldb):
     """Test for the 1st globalDB data migration"""
     # Check state before migration
     with globaldb.conn.read_ctx() as cursor:
-        assert globaldb.get_setting_value('version', None) == 6
+        assert globaldb.get_setting_value('version', None) == 7
```

Comment:
If this needs to be always the latest version change it to the constant `GLOBAL_DB_VERSION`